### PR TITLE
Update bernardofoegbu.json

### PR DESCRIPTION
**Pull request by O-BERNARDOFOEGBU**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

my site is already up " https://bernardofoegbu.is-a.dev "
I am trying indexing and optimizing my website and I needed to Verify domain ownership via DNS record and I need to add a txt record from google which is to be verified.

### CODE REVIEW

File: domains/bernardofoegbu.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
     "email": "obinnaofoegbu71@gmail.com"
   },
   "record": {
-    "CNAME": "o-bernardofoegbu.github.io"
+    "CNAME": "o-bernardofoegbu.github.io",
```
Comment (Stef-00012) [inline domains/bernardofoegbu.json:9]: You can not use CNAME rexords together with any other record

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/bernardofoegbu.json (hunk 1 above)
Comment (O-BERNARDOFOEGBU) [reply domains/bernardofoegbu.json]: okay... will use just the txt

**Review by O-BERNARDOFOEGBU — COMMENTED** [review]
sorted

File: domains/bernardofoegbu.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
     "email": "obinnaofoegbu71@gmail.com"
   },
   "record": {
-    "CNAME": "o-bernardofoegbu.github.io"
+     "CNAME": "o-bernardofoegbu.github.io",
+     "CNAME_Label_Host": "bdvmkvxl5dlz",
+     "CNAME_Destination_Target": "gv-yrwqiastbtprak.dv.googlehosted.com"
```
Comment (orxngc) [inline domains/bernardofoegbu.json:11]: ... these are not valid record types, 
please read our [documentation](https://is-a.dev/docs)

File: domains/bernardofoegbu.json [hunk 3]
```diff
@@ -6,6 +6,6 @@
     "email": "obinnaofoegbu71@gmail.com"
   },
   "record": {
-    "CNAME": "o-bernardofoegbu.github.io"
+     "TXT": "google-site-verification=VAoiuOjvQxUCsqxyZUGS9WqMLVi2lSxOAnmyENy0Sq4"
```
Comment (DEV-DIBSTER) [inline domains/bernardofoegbu.json:9]: Should mention this. TXT records are used for verification, but when they're not paired with any other records, they're pretty useless. You should add the other records that Google requires.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**